### ISSUE TITLE
JGET multi file query

### ISSUE THREAD

**Issue by jack-mei** [tidwall/tile38#284]
The values set through the JSET can wake up by waking up to update why the value of the query label is not provided through the JGE

**Comment by tidwall** [comment]
Hi @jack-mei, 
Can you provide an example of the problem?

**Comment by jack-mei** [comment]
For example, Key-Value is stored through JSET, Value is a JSON file, and it needs to query through a JSON file node value, which contains key of the node value in the entire storage set.

**Comment by tidwall** [comment]
I'm sorry but I don't understand the problem. Could you provide an example of what commands you are sending Tile38, what results Tile38 is returning, and what you expect should happen?

**Comment by jack-mei** [comment]
Oh, I'm sorry. Maybe I didn't describe it clearly. Can I give you a distance description in a relational database?

**Comment by tidwall** [comment]
No, please provide the Tile38 commands. Include the result. And what you expect should happen.

**Comment by jack-mei** [comment]
Ok， Wait a moment

**Comment by jack-mei** [comment]
Data Base TABLE:
ID | NAME | AGE | SEX | ADDRESS
-- | -- | -- | -- | --
1710231744170005 | Peki | 8 | girl | 杨桥湖大道N097
1156221023174417 | George | 5 | boy | 杨桥湖大道N093
1710231744173445 | John | 8 | boy | 杨桥湖大道N095
1710231744170123 | Jack | 7 | boy | 杨桥湖大道N096

For example, query in a relational database  "SELECT  ID  FROM TABLE WHERE AGE = '8' "  
The query result has two information  "1710231744170005 " and "1710231744173445 "

So.  Key-Value is stored through JSET, Value is a JSON file,
For example,
objects: [
{
id: "1710231744170005",
object: "{"properties":"NAME":"Peki","AGE":"8","SEX":"girl","address":"杨桥湖大道N097"}}"
 },

id: "1156221023174417 ",
object: "{"properties":"NAME":"George","AGE":"5","SEX":"boy","address":"杨桥湖大道N093"}}"
 },
 id: "1710231744173445",
object: "{"properties":"NAME":"John","AGE":"8","SEX":"girl","address":"杨桥湖大道N095"}}"
 },
 id: "1710231744170123",
object: "{"properties":"NAME":"Jack","AGE":"7","SEX":"boy","address":"杨桥湖大道N096"}}"
 }
]

I want to get the same query results as the SQL statement， where AGE = 8.
Can tile38 provide an interface like this

**Comment by jack-mei** [comment]
I don't know if you know what I mean, thank you

**Comment by tidwall** [comment]
Sorry but Tile38 is not a relational database.

**Comment by jack-mei** [comment]
Do you team, will it be possible to consider such a similar interface in the future

**Comment by tidwall** [comment]
It may be possible in the future. Check out [SummitDB](https://github.com/tidwall/summitdb). It's more of what you are looking for.
